backend/socket.io/sync.js

```diff
@@ -1,5 +1,13 @@
 var state = require("./state.json");
 
+const feedbackOptions = {
+  APPROVE: "Your code is approved! 👍 Now it's waiting for a drone to fly...",
+  ALMOST_THERE:
+    "Your code is pretty good but there's still something small that you need to change. Have a look! 😃 ",
```

Comment:
"... there's still some small things you need to change." sounds a bit more natural